test/files/jvm/patmat_opt_no_nullcheck.check

```diff
@@ -1 +1,26 @@
-bytecode identical
```

Comment:
Argh, sorry, this was not intentional!

If I'm not mistaken, I saw that there is another method in the ASM API, something along the lines of "semantically equal". I'll try that before moving it to pending.